modules/jfx.incubator.richtext/src/main/java/com/sun/jfx/incubator/scene/control/richtext/StyleAttributeMapHelper.java

```diff
@@ -52,20 +52,39 @@ public static void setAccessor(Accessor a) {
     /**
      * Returns a new StyleAttributeMap instance which contains only the character attributes,
      * or null if no character attributes found.
+     *
      * @param ss the style attribute map
-     * @return the instance
+     * @return the instance of StyleAttributeMap, or null
```

Comment:
Is there a good reason to allow null here? Unless there is a semantic difference between null and the empty map, it might be easier to make this non-nullable. This could be done later, since the fact that it can return null is preexisting.

Reply:
I've recorded this in the feedback document - this might be worth doing.